test/e2e/select.test.js

```diff
@@ -142,6 +141,49 @@ describe('vl-select', async () => {
     await assert.eventually.isTrue(select.hasValue('two'));
   });
 
+  it('als gebruiker kan ik een select dynamisch activeren met gegroepeerde opties', async () => {
```

Comment:
Er wordt ook gecontroleerd dat een optie gekozen kan worden, eventueel verduidelijken in de test beschrijving.

Reply:
testomschrijving aangepast